DBAL/TracingDriverConnection.php

```diff
@@ -13,10 +13,10 @@
  */
 final class TracingDriverConnection implements DBALDriverConnection, WrappingDriverConnection
 {
-    private $decoratedConnection;
-    private $tracing;
-    private $spanFactory;
-    private $username;
+    private DBALDriverConnection $decoratedConnection;
+    private Tracing $tracing;
+    private SpanFactory $spanFactory;
+    private ?string $username = null;
```

Comment:
There should be no need to default it with null since it will be overwritten on construction.

Reply:
True, I will change that.